### PR TITLE
Add momoka ergo via support

### PR DESCRIPTION
**Pull request by kevinboss**
## Description

Add keymap with via support for the Momoka Ergo https://momoka.store/products/momoka-ergo

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/momoka_ergo/keymaps/via/rules.mk [hunk 1]
```diff
@@ -0,0 +1,9 @@
+VIA_ENABLE = yes
```
Comment (kevinboss) [inline keyboards/momoka_ergo/keymaps/via/rules.mk:1]: It says to not include makefiles into the keymap, but I don't wish to toggle these flags for the default keymap. How do I go about that?

**Comment by kevinboss** [comment]
The functionality can be tested by using https://github.com/kevinboss/the-via-keyboards/blob/feature/momoka_ergo/src/momoka/ergo/ergo.json

File: keyboards/momoka_ergo/keymaps/via/rules.mk (hunk 1 above)
Comment (fauxpark) [reply keyboards/momoka_ergo/keymaps/via/rules.mk:1]: That is talking about an actual file named `Makefile`, not `rules.mk`.

File: keyboards/momoka_ergo/keymaps/via/rules.mk [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+LTO_ENABLE = yes
+LEADER_ENABLE = no
+FAUXCLICKY_ENABLE = no
+MIDI_ENABLE = no
+BLUETOOTH_ENABLE = no
+KEY_LOCK_ENABLE = no
+TERMINAL_ENABLE = no
+BOOTMAGIC_ENABLE = lite
```
Comment (fauxpark) [inline keyboards/momoka_ergo/keymaps/via/rules.mk:9]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/momoka_ergo/keymaps/via/rules.mk (hunk 2 above)
Comment (kevinboss) [reply keyboards/momoka_ergo/keymaps/via/rules.mk:9]: @fauxpark Just so I understand, this isn't necessary?
I was following the recommendation here: https://www.caniusevia.com/docs/configuring_qmk/#create-a-rulesmk-in-keyboardskeyboard_namekeymapsvia

Comment (fauxpark) [reply keyboards/momoka_ergo/keymaps/via/rules.mk:9]: That documentation is outdated; those features no longer change the keycode values. However it's not necessary to add them anyway if the keyboard-level rules.mk does not set them to `yes`, as they effectively default to `no`.

Bootmagic Full has been removed, and Lite is now the default, so `lite` is not a valid value here. Additionally, `VIA_ENABLE = yes` implies `BOOTMAGIC_ENABLE = yes` so there is no need to explicitly enable it.

**Review by fauxpark — APPROVED** [review]

**Review by keyboard-magpie — APPROVED** [review]

**Pull request merged**